scripts/release/inline-resources.js

```diff
@@ -93,7 +93,9 @@ function inlineTemplate(content, urlResolver) {
     const templateContent = fs.readFileSync(templateFile, 'utf-8');
     const shortenedTemplate = templateContent
       .replace(/([\n\r]\s*)+/gm, ' ')
```

Comment:
Wouldn't that remove the spaces between attributes too aggressively? Such as after `<ng-content`:
```html
<ng-content select="[md-card-avatar], [mat-card-avatar]"></ng-content>
<div class="mat-card-header-text">
  <ng-content
      select="md-card-title, mat-card-title, md-card-subtitle, mat-card-subtitle,
      [md-card-title], [mat-card-title], [md-card-subtitle], [mat-card-subtitle]"></ng-content>
</div>
<ng-content></ng-content>
```